### PR TITLE
HTS-1158 Added tests to verify titles and headers

### PR DESCRIPTION
**Pull request by Jackie1988**

### CODE REVIEW

File: selenium-system-test/src/test/resources/features/accountHolder/Account holder changes their email.feature [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Feature: Account holder changes their email
 
-  @HTS-876
+  @HTS-876 @VerifyHandF
```
Comment (MarcusQA) [inline selenium-system-test/src/test/resources/features/accountHolder/Account holder changes their email.feature]: @Jackie1988 Can we change the tag to be @VerifyHeaderAndFooter, please?

File: selenium-system-test/src/test/resources/features/common/Verify header and footer links.feature [hunk 2]
```diff
@@ -1,6 +1,30 @@
 Feature: Verifying header and footer links on every page
 
-  @HTS-668
+  @HTS-668 @VerifyHandF
   Scenario: when users navigates through the HTS pages they see feedback, get-help and privacy links are working
     When the user has logged in and passed IV
     Then they go through the happy path they can see and access all header and footer links
+
+  @VerifyHandF
+  Scenario: checking the Header and Footer links of the pages that have not been tested so far when logged in
```
Comment (MarcusQA) [inline selenium-system-test/src/test/resources/features/common/Verify header and footer links.feature]: @Jackie1988 Is there a more specific scenario name we can give for these pages?

File: selenium-system-test/src/test/resources/features/accountHolder/Account holder changes their email.feature (hunk 1 above)
Comment (Jackie1988) [reply selenium-system-test/src/test/resources/features/accountHolder/Account holder changes their email.feature]: ok done thanks

File: selenium-system-test/src/test/resources/features/common/Verify header and footer links.feature (hunk 2 above)
Comment (Jackie1988) [reply selenium-system-test/src/test/resources/features/common/Verify header and footer links.feature]: yeah sure

**Pull request merged**